maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/report/PluginReport.java

```diff
@@ -159,12 +159,42 @@
     /**
      * Set this to "true" to generate the usage section for "plugin-info.html" with
      * {@code <extensions>true</extensions>}.
-     * 
+     *
      * @since 3.7.0
      */
     @Parameter( defaultValue = "false", property = "maven.plugin.report.hasExtensionsToLoad" )
     private boolean hasExtensionsToLoad;
 
+    /**
+     * The Plugin requirements history list.
+     * <p>
+     * Can be specified as list of string in format:
+     *
+     * <pre>
+     * &lt;requirementsHistories&gt;
+     *   &lt;requirementsHistory&gt;
+     *     plugin-version requires Maven maven-version and JDK jdk-version
+     *   &lt;/requirementsHistory&gt;
+     * &lt;/requirementsHistories&gt;
+     * </pre>
+     *
+     * or as xml:
+     *
+     * <pre>
+     * &lt;requirementsHistories&gt;
+     *   &lt;requirementsHistory&gt;
+     *     &lt;version&gt;plugin version&lt;/version&gt;
+     *     &lt;maven&gt;maven version&lt;/maven&gt;
+     *     &lt;jdk&gt;jdk version&lt;/jdk&gt;
+     *   &lt;/requirementsHistory&gt;
+     * &lt;/requirementsHistories&gt;
```

Comment:
I wouldn't support both, it just causes confusion and inconsistency. I'd go with the second approach.